templates/migrations/file_upload_endpoints.txt

```diff
@@ -44,7 +44,7 @@ export default class FileUploadEndpoints extends BaseSchema {
       table.string('accept', 2048).comment('comma-delimited string of accepted file extensions.')
       table.boolean('multiple_files_enabled').notNullable().defaultTo(true)
       table
-        .enu('storage_type', ['local', 's3'], { useNative: true, enumName: 'enu_po_status' })
+        .enu('storage_type', ['local', 's3'], { useNative: false, enumName: 'enu_po_status' })
```

Comment:
Do you know why we have "po_status" as the enum name?  Can we rename to `enu_storage_types` ?  I'm guessing it was a copy/paste thing from another project.